cluster/manifests/skipper/deployment.yaml

```diff
@@ -483,6 +484,7 @@ spec:
 {{ if or (eq .ConfigItems.nlb_switch "pre") (eq .ConfigItems.nlb_switch "exec") }}
           - "-forwarded-headers=X-Forwarded-For,X-Forwarded-Proto=https,X-Forwarded-Port=443"
```

Comment:
This enforces `X-Forwarded-Proto: https` behind NLB but only except `-forwarded-headers-exclude-cidrs`.
Redirect routes never match such requests as they use `Header("X-Forwarded-Proto", "http")` predicate.